hc/accounts/admin.py

```diff
@@ -119,6 +120,7 @@ def engagement(self, user):
 
     engagement.allow_tags = True
```

Comment:
With the `@mark_safe` annotation, looks like this and `checks.allow_tags = True` is not needed any more.

Reply:
You're right, it looks like Allow tags support has been removed in Django 2.0